assets/src/components/TableAssignment.js

```diff
@@ -140,10 +151,33 @@ function CustomAssignmentTable (props) {
           if (isCurrentWeek) {
             borderAndCurrentWeekStyle['color'] = 'orange'
           }
+          // styling for assignment title column
+          if (j === 2) {
+            borderAndCurrentWeekStyle['width'] = '30%'
+            borderAndCurrentWeekStyle['maxWidth'] = 0
+            borderAndCurrentWeekStyle['overflow'] = 'hidden'
+            borderAndCurrentWeekStyle['textOverflow'] = 'ellipsis'
+            borderAndCurrentWeekStyle['whiteSpace'] = 'nowrap'
+            borderAndCurrentWeekStyle['borderRight'] = '0.1em solid rgba(224, 224, 224, 1)'
+          }
+          // for week/due_date column don't need much space as the text length is fixed
+          if (j < 1) {
+            borderAndCurrentWeekStyle['width'] = '10%'
+          }
+          // for the horizontal bar we want the bar touching the line
+          if (j === 3) {
+            borderAndCurrentWeekStyle['paddingLeft'] = 0
+          }
           return (
-            <TableCell className={classes.tableCell} key={j} style={borderAndCurrentWeekStyle}>
-              {displayProp ? prop : null}
-            </TableCell>
+            j === 2 ?
+              <TableCell title={displayProp ? prop : null} className={classes.tableCell} key={j}
+                         style={borderAndCurrentWeekStyle}>
```

Comment:
since long assignment get truncated user is given the option on hover to review the full assignment time. Well, this feature in mobile is not able to see the full title as the title seems to work only with hover and not with on click on mobile. Another option is adding the CSS tooltip, thoughts???